course-mw/enrolment-actor/src/main/scala/org/sunbird/enrolments/CourseEnrolmentActor.scala

```diff
@@ -313,10 +315,10 @@ class CourseEnrolmentActor @Inject()(@Named("course-batch-notification-actor") c
         }
     }
 
-    def getEnrolmentList(request: Request, userId: String): Response = {
+    def getEnrolmentList(request: Request, userId: String, courseIdList: java.util.List[String]): Response = {
```

Comment:
can we change this for properties which can include any other filtering too like batch id? 
That can help to make the code reusable.

Reply:
Will be addressed in Ticket # https://project-sunbird.atlassian.net/browse/SB-29749